src/Microsoft.DotNet.CodeFormatting.Tests/FormatOptionsParsingTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+
+using CodeFormatter;
+
+using CommandLine;
+
+using Xunit;
+
+namespace Microsoft.DotNet.CodeFormatting.Tests
+{
+    public class FormatOptionsParsingTests
+    {
+        private FormatOptions Parse(out int result, params string[] args)
+        {
+            result = 1;
+            FormatOptions options = null;
+
+            using (var writer = new StringWriter())
```

Comment:
yes, good catch, vestiges of a broken attempt to fix a test multi-threading issue